rocksdb_admin/admin_handler.cpp

```diff
@@ -160,6 +168,8 @@ const std::string kHDFSBackupMs = "hdfs_backup_ms";
 const std::string kHDFSRestoreMs = "hdfs_restore_ms";
 const std::string kS3BackupMs = "s3_backup_ms";
 const std::string kS3RestoreMs = "s3_restore_ms";
+const std::string kDeleteDBFailure = "delete_db_failure";
+const int kDeleteDBWaitSec = 80;
```

Comment:
it's better to make it a gflag so that we can tune without building a new package.